PepperDashEssentials/ControlSystem.cs

```diff
@@ -131,29 +131,46 @@ public void DeterminePlatform()
 
                 if (CrestronEnvironment.DevicePlatform != eDevicePlatform.Server)   // Handles 3-series running Windows CE OS
                 {
-                    Debug.Console(0, Debug.ErrorLogLevel.Notice, "Starting Essentials v{0} on {1} Appliance", Global.AssemblyVersion, Global.ProcessorSeries.ToString());
+                    string userFolder;

```

Comment:
Really, we can set the folder names to lower-case in ALL situations and we should be fine. 3-Series isn't case-sensitive cause Windows.